src/parts/Sidebar/SidebarContent/Navigation/index.tsx

```diff
@@ -88,7 +90,7 @@ const Navigation = ({
       },
       {
         name: 'nav_vaults',
-        link: `${PAGES.VAULTS.replace(`:${URL_PARAMETERS.VAULT.ACCOUNT}`, address)}`,
+        link: `${PAGES.VAULTS.replace(`:${URL_PARAMETERS.VAULT.ACCOUNT}`, selectedAccount?.address ?? '')}`,
```

Comment:
Correct me if im wrong. At some point, this will be an invalid url. We can still memoize this when doing the following:
```
if(selectedAccount?.address) {
items.push({...});
}
```

Reply:
We have a potential ticket around this so let's handle that at that time.